PhilzAce1/GEO-SEARCH-
# ch-responsive-ui-29518363

# ch-responsive-ui-29518363
#### What does this PR do?
A responsive view 
#### Description of Task to be completed?
Responsiveness working 
#### How should this be manually tested?
After cloning and Installing dependencies 
open src/index with live server 
check responsiveness with chrome developer tools
#### Any background context you want to provide?
some gulp plugins may be deprecated 
#### What are the relevant github board stories?
29518363
#### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/46319584/69284812-c6e1d680-0bef-11ea-9d15-ae58711cf7b9.png)
![image](https://user-images.githubusercontent.com/46319584/69284879-ee38a380-0bef-11ea-907e-435e3b2c1ccd.png)
